### ISSUE TITLE
Load Average Is Per Core By Default Contrary to the Docs

### ISSUE THREAD

**Issue by AaronKalair** [sensu-plugins/sensu-plugins-load-checks#18]
Hello,

The documentation in this file - https://github.com/sensu-plugins/sensu-plugins-load-checks/blob/master/bin/metrics-load.rb#L12 

Says that 

```
#   Optionally, with `--per-core`, this plugin will calculate load per
#   processor from the raw load average by dividing load average by the number
#   of processors.
```

However the --per-core flag is never used in the code, and by default the load average is emitted per core. 

Is the issue that this flag needs to be implemented (making it a backwards incompatible change if people update the plugin) or should there be a `--not-per-core` that reports the overall load average?

**Comment by majormoses** [comment]
Thanks for reporting this, this was removed/changed in https://github.com/sensu-plugins/sensu-plugins-load-checks/pull/15 for a multitude of issues + confusion that plauged this plugin. I would be happy to take a PR to update the description in the plugin to match. Sorry for the confusion!

**Comment by majormoses** [comment]
closed via #20
